src/Api/Vault/AuthorizationHandlers/Collections/BulkCollectionAuthorizationHandler.cs

```diff
@@ -247,19 +257,32 @@ public class BulkCollectionAuthorizationHandler : BulkAuthorizationHandler<BulkC
 
     private async Task<bool> IsAssignedToCollectionsAsync(
         ICollection<Collection> targetCollections,
-        CurrentContextOrganization org,
+        CurrentContextOrganization? org,
```

Comment:
The org shouldn't be nullable by this point. (If the org is null, they're not a member and there's no point doing the db lookup.)

Did you hit any errors with it not being nullable, or are you just addressing IDE suggestions? I'd rather leave it not nullable here and ensure that callers check for null or `HasValue` first.

(As an aside - we use a lot of tricky pattern matching just to avoid explicit null checks. I wonder whether it's easier to just check for null - but I don't actually know.)

Reply:
I wasn't 100% sure on this. The reason its even questionable is because technically from the `CurrentContext` the method `GetOrganization` might not have the organization within the claims, which would return `null`. However, I've reviewed the change and to match the flow of the other requirement checks, I've added a `null` check for the org in that `CanDeleteAsync` conditional and reverted the optional operator from the `IsAssignedToCollectionsAsync` signature and method.